### ISSUE TITLE
Do not open external links in new tabs

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#496]
This issue was modified as a result of the discussion. Refer to the edit history for previous versions and context on the discussion up until https://github.com/WordPress/openverse/issues/496#issuecomment-2147281933.

## Problem

Get this audio link opens in a new tab, but does not disclose this to screen readers.

## Description

As discussed, we will no longer open links in new tabs. Keep the external site icon, but remove the new tab behaviour from an links that currently do this. Review the behaviour of `VLink` and any usage of `target="_blank"` in the frontend.

Update or add e2e tests that evaluate this behaviour to ensure new tabs are not opened from external links in practice. It might be a good opportunity to add a functionality test for VLink in Storybook, if that appropriately reflects an aspect of how this is solved in the code.

## Additional context

This was originally reported for the `Get audio` external link, by Joe Dolson (https://make.wordpress.org/openverse/2022/09/07/frontend-release-v3-4-8-and-a-call-for-a11y-testing/#comment-95)

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by dhruvkb** [comment]
Can we reconsider if we should be opening new tabs at all instead?

The W3C guidelines [recommend not opening links in new tabs](https://www.w3.org/TR/WCAG20-TECHS/G200.html) unless it is absolutely necessary (with some examples about what counts as necessary). They also recommend clearly stating if a link opens a new tab, which I feel is not adequately conveyed by just the external site icon.

Our external links could open in the same tab and let users who want new tabs do so via the context menu or holding down <kbd>Ctrl</kbd> or <kbd>Cmd</kbd>.

**Comment by kvdotkirti** [comment]
Is this issue still open? I would like to attempt it

**Comment by obulat** [comment]
> Can we reconsider if we should be opening new tabs at all instead?

Thank you for pointing this out. I think we _should stop_ opening external links in new tabs. 
Should we still have an external link icon next to links that lead to external websites? That is the behavior on WordPress.org: the external links have an icon, but open in the same tab.
@zackkrida, @fcoveram, @sarayourfriend, what do you think?

I removed the "help wanted" labels until this discussion is finalized.

**Comment by fcoveram** [comment]
Agree with @dhruvkb. Let's not open new tabs

**Comment by obulat** [comment]
> Agree with @dhruvkb. Let's not open new tabs

Do you think we should still have the "External link" icon, @fcoveram?

**Comment by fcoveram** [comment]
Oh yes. Sorry for not being clear: Keep the external action (leaving the site from a link with the external icon) but in the same tab, not in a new one.

**Comment by sarayourfriend** [comment]
I've updated the issue to reflect the decision. I won't add good first issue because it will require a bit of searching around the app to make sure it is handled in all cases as well as probably updating or add e2e tests to reflect the change.
